drivers/wifi/winc1500/wifi_winc1500.c

```diff
@@ -48,7 +48,7 @@ NMI_API sint8 listen(SOCKET sock, uint8 backlog);
 NMI_API sint8 accept(SOCKET sock, struct sockaddr *addr, uint8 *addrlen);
 NMI_API sint8 connect(SOCKET sock, struct sockaddr *pstrAddr, uint8 u8AddrLen);
 NMI_API sint16 recv(SOCKET sock, void *pvRecvBuf,
-		    uint16 u16BufLen, uint32 u32Timeoutmsec);
+		    uint16 u16BufLen, k_timeout_t u32Timeoutmsec);
```

Comment:
How is this going to work as in `hal/atmel/asf/common/components/wifi/winc1500/socket/include/socket.h` the recv() is defined to be `NMI_API sint16 recv(SOCKET sock, void *pvRecvBuf, uint16 u16BufLen, uint32 u32Timeoutmsec);`
It would be much simpler if you keep the msec timeout as is, and just convert to to k_timeout_t when absolutely needed (like in a call to `k_sem_take()`

Reply:
I see your point. I will get it fixe.

Reply:
Do you think I need to make another pull request to get my new changes in?

Reply:
> Do you think I need to make another pull request to get my new changes in?

No, using this PR is just fine.